core-web/apps/dotcms-ui/src/app/portlets/dot-containers/container-list/container-list.component.ts

```diff
@@ -59,4 +79,54 @@ export class ContainerListComponent {
     resetBundleIdentifier(): void {
         this.store.updateBundleIdentifier(null);
     }
+
+    setContainerActions(container: DotContainer): DotActionMenuItem[] {
+        return this.store.getContainerActions(container);
+    }
+
+    private notifyResult(response: DotActionBulkResult | DotContainer, messageKey: string): void {
+        if ('fails' in response && response.fails.length) {
+            this.showErrorDialog({
+                ...response,
+                fails: this.getFailsInfo(response.fails),
+                action: this.dotMessageService.get(messageKey)
+            });
+        } else {
+            this.showToastNotification(this.dotMessageService.get(messageKey));
+        }
```

Comment:
Why don't you just encapsulate all this logic in the store and just send something to indicate if sucess or fail with their respective messages keys resolved?